### PR TITLE
ELEMENTS-1312: simplify nuxeo-path-suggestion validation

### PR DESCRIPTION
**Pull request by richardsd**

### CODE REVIEW

**Comment by richardsd** [comment]
Pipeline https://qa2.nuxeo.org/jenkins/job/WebUI/job/web-ui-pipeline-10.10/728/ 🕐

File: nuxeo-document-preview.html [hunk 1]
```diff
@@ -253,13 +253,11 @@
...
-              .filter(
-                (conversion) =>
-                  conversion &&
+              .filter((conversion) =>
+                conversion &&
                   conversion.content &&
                   conversion.content.data &&
-                  conversion.content['mime-type'].match(/^video.*/),
-              )
+                  conversion.content['mime-type'].match(/^video.*/))
```
Comment (mnixo) [inline nuxeo-document-preview.html:260]: An indentation level was removed above, we can have all conditions on the same level:
```
conversion &&
conversion.content &&
conversion.content.data &&
conversion.content['mime-type'].match(/^video.*/))
```

**Review by mnixo — DISMISSED** [review]

File: nuxeo-document-preview.html (hunk 1 above)
Comment (richardsd) [reply nuxeo-document-preview.html:260]: We can, but it wasn't me changing it. I used `eslint` with `--fix` option to solve the linting issues 🤷

Comment (richardsd) [reply nuxeo-document-preview.html]: Done 👍 Thanks

**Review by mnixo — APPROVED** [review]

**Review by Gabez0r — APPROVED** [review]
LGTM. @richardsd, please make sure this passes on the pipeline.

**Comment by richardsd** [comment]
> LGTM. @richardsd, please make sure this passes on the pipeline.

It was already here qa2.nuxeo.org/jenkins/job/WebUI/job/web-ui-pipeline-10.10/731 🔵

**Pull request merged**